subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AbstractSmokeTest.groovy

```diff
@@ -214,22 +214,6 @@ abstract class AbstractSmokeTest extends Specification {
             }
         }
 
-        /**
-         * Since Android 7.3.0 is not yet stable we have to use that.
-         * One stable version is released we should remove this.
-         */
-        String latestStableOrRc() {
-            def stableVersion = latestStable()
-            if (stableVersion != null) {
-                return stableVersion
-            }
-            return versions.reverse().find { version ->
-                    !version.containsIgnoreCase("beta") &&
-                    !version.containsIgnoreCase("alpha") &&
-                    !version.containsIgnoreCase("milestone")
-            }
-        }
```

Comment:
Would this come in handy for 7.4 RCs?

Reply:
I think not, since we normally use latest stable version in this test. This was added just for 7.3.0-rc, since there was no other stable version that would work with Gradle 8.0.

Reply:
I will merge it, if you think this method would come handy let me know.